epidose/device/wps_scanner_d.sh

```diff
@@ -203,7 +203,10 @@ while : ; do
   if try_alternative_wifi_connections; then
     # Stop the sleep process of update_filter_d to get a new filter
     log "Killing update_filter_d's sleep process"
-    kill -9 "$(cat "$SLEEP_UPDATE_FILTER_PID")"
+    # Catch non-zero exitcode to avoid daemon crush
```

Comment:
Good catch!  BTW, it's crash, not crush.

Reply:
Ok, I will submit a fix on this.